src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -2820,6 +2820,15 @@ void mvnw(Register Rd, Register Rm,
     f(0b000001, 15, 10), rf(Vn, 5), rf(Vd, 0);
   }
 
+  // Advanced SIMD scalar copy
+  void dups(FloatRegister Vd, SIMD_Arrangement T, FloatRegister Vn, int index = 0)
+  {
+    starti;
+    f(0b01011110000, 31, 21);
+    f(((1 << (T >> 1)) | (index << ((T >> 1) + 1))), 20, 16);
+    f(0b000001, 15, 10), rf(Vn, 5), rf(Vd, 0);
+  }
+
```

Comment:
One is _Advanced SIMD copy_, the other _Advanced SIMD scalar copy_.  The _Advanced SIMD copy_ version is missing a comment to that effect. Maybe one day someone could go through assembler_aarch64.hpp and add the missing titles from the A64 instruction set encoding section of the ARM.

Reply:
> One is _Advanced SIMD copy_, the other _Advanced SIMD scalar copy_. The _Advanced SIMD copy_ version is missing a comment to that effect. Maybe one day someone could go through assembler_aarch64.hpp and add the missing titles from the A64 instruction set encoding section of the ARM.

Yes, a comment will be helpful. But for the scalar copy, we should use SIMD_RegVariant instead of SIMD_Arrangement?

Reply:
It looks like the official doc uses _SIMD_RegVariant_ for the scalar copy, so we should do the same. I don't think you'll ever find me suggesting we should deviate from what the Arm reference manuals do, unless we can't avoid it.

Reply:
> Yes, a comment will be helpful. But for the scalar copy, we should use SIMD_RegVariant instead of SIMD_Arrangement?

Thanks, we have fixed it.